src/containers/AssignmentTexterContact.jsx

```diff
@@ -172,7 +172,7 @@ const inlineStyles = {
   }
 }
 
-class AssignmentTexterContact extends React.Component {
+export class AssignmentTexterContact extends React.Component {
```

Comment:
Did you add this to make possible for testing?

Reply:
yes